### ISSUE TITLE
subdaily dataset - Date time format

### ISSUE THREAD

**Issue by rposn** [StatisticalServicesCentre/ClimateObject#18]
I am working with a subdaily dataset. My understanding is that the method "date_col_check" checks first whether the "date_col" a Date class is and, if not, then it will convert it to a Date class (see line 539 of the first picture attached), before going to the section referred by @dannyparsons . Then the time information is already removed from the date column before getting to the section referring to the "subdaily_label" (second picture attached)
![date_col_check](https://cloud.githubusercontent.com/assets/12081199/7816292/a0cf1ad6-03cd-11e5-8ec6-cc155bda52d8.PNG)
![984aacc8-03ae-11e5-9477-739f08ca1c0b](https://cloud.githubusercontent.com/assets/12081199/7816346/1573cc2e-03ce-11e5-94e7-106ac81d1c5b.jpg)

**Comment by volloholic** [comment]
You're absolutely right there was a problem with the ordering here. Some of the main issue have now been addressed but not checked. I believe that date_col_check should now accept subdaily data better but we need to discuss how it will be passed in to get the defaults working better.

**Comment by rposn** [comment]
After trying the last version of the "date_col_check" method I received the Warning message:  
"Cannot create or edit a date column. There is insufficient information in the data frame to have a date column."
Therefore I modified the method by adding the a new if statement in line 625. In this way the climObject worked for me for "subdaily" datasets.
Please check whether this makes sense. Thanks
PS. commit of the change: f9b523087166941833a8dc6168aee34d35a096cc

**Comment by volloholic** [comment]
Hi I'd just made a correction of that problem in a different way. I hope the latest version will work for you otherwise let's has a quick share screen to sort this out.
